LoopKit/DosingDecisionStore.swift

```diff
@@ -26,22 +26,20 @@ public class DosingDecisionStore {
     private let store: PersistenceController
     private let expireAfter: TimeInterval
     private let dataAccessQueue = DispatchQueue(label: "com.loopkit.DosingDecisionStore.dataAccessQueue", qos: .utility)
-    private let log = OSLog(category: "DosingDecisionStore")
+    public let log = OSLog(category: "DosingDecisionStore")
```

Comment:
[nit] Was it necessary to make this public?  I don't see where you needed to...

Reply:
There is an extension of `DosingDecisionStore` in `Loop` that uses the log.